code/src/App.js

```diff
@@ -1,9 +1,124 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './app.css'
 
 export const App = () => {
+  const [destination, setDestination] = useState('beach')
+  const [type, setType] = useState('romantic')
+  const [budget, setBudget] = useState('low-cost')
+  const [summary, setSummary] = useState(false)
+  const [question, setQuestion] = useState('destination')
+
+  const submitForm = (event) => {
+    event.preventDefault()
+    setSummary(true)
+  }
+
+  const nextQuestion = () => {
+    if (question === 'destination') {
+      setQuestion('type')
+    } else if (question === 'type') {
+      setQuestion('budget')
+    }
+  }
+
   return (
     <div>
-      Find me in src/app.js!
+      {summary && (
+        <div className="summary">
+          <h1>Got it!</h1>
+          <p>We are taking you in {budget === 'low-cost' ? 'a ' + budget : 'an ' + budget} {type === 'familyFriendly' ? 'family friendly' : type} experience to {destination === 'beach' ? 'sandy beaches' : 'the ' + destination}!</p>
+        </div>
+
+      )}
+
+      {!summary && (
+        <form onSubmit={submitForm}>
+
+          {question === 'destination' && (
+            <div className="destination">
+              <p>Where do you wanna go?</p>
+              <label>
+                <input
+                  type="radio"
+                  value="beach"
+                  onChange={() => setDestination('beach')}
+                  checked={destination === 'beach'} />
+                <span>Beach</span>
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  value="countryside"
+                  onChange={() => setDestination('countryside')}
+                  checked={destination === 'countryside'} />
+                <span>Countryside</span>
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  value="city"
+                  onChange={() => setDestination('city')}
+                  checked={destination === 'city'} />
+                <span>City</span>
+              </label>
+            </div>
+          )}
+
+          {question === 'type' && (
+            <div className="type">
+              <p>What kind of trip you're looking for?</p>
+              <label>
```

Comment:
If you want to improve this you could try the .map() solution. But it's not necessary!